float64 target_angle

string result

float64 current_z_angle
string debug_msg
